app/_src/policies/meshcircuitbreaker.md

```diff
@@ -591,6 +621,46 @@ spec:
     kind: MeshSubset
     tags:
       app: web
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        outlierDetection:
+          interval: 5s
+          baseEjectionTime: 30s
+          maxEjectionPercent: 20
+          splitExternalAndLocalErrors: true
+          detectors:
+          detectors:
+            totalFailures:
+              consecutive: 10
+            gatewayFailures:
+              consecutive: 10
+            localOriginFailures:
+              consecutive: 10
+            successRate:
+              minimumHosts: 5
+              requestVolume: 10
+              standardDeviationFactor: 1.9
+            failurePercentage:
+              requestVolume: 10
+              minimumHosts: 5
+              threshold: 85
+```
+{% endpolicy_yaml %}
+{% endif_version %}
+
+{% if_version gte:2.10.x %}
+{% policy_yaml protocol-210x namespace=kuma-demo %}
+```yaml
+type: MeshCircuitBreaker
+name: backend-inbound-outlier-detection
+mesh: default
+spec:
+  targetRef:
+    kind: Dataplane
+    labels:
+      app: web
```

Comment:
should we use rules here?

Reply:
I will, but I started with just changing docs for dataplane kind. We have separate issues for `rules` api